backend/src/controllers/userController.ts

```diff
@@ -0,0 +1,82 @@
+import { PrismaClient } from "@prisma/client";
+import { Request, Response } from "express";
+import { generateNonce } from "siwe";
+import { errorResponse } from "../utils/utils";
+
+const prisma = new PrismaClient();
+
+/**
+ *
+ * @param req
+ * @param res
+ * @returns
+ */
+const createUser = async (req: Request, res: Response) => {
+  const { address } = req.params;
+  const nonce = generateNonce();
+
+  try {
+    if (address) {
+      await prisma.user.create({
+        data: {
+          address,
+          nonce,
+        },
+      });
+
```

Comment:
You are not checking if the user existe or not. 
And you are not cheking if who whant to create a user for this address is the owner of the wallet.

Reply:
Okay, noted about that